pyproject.toml

```diff
@@ -18,8 +18,8 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
-    "argilla >= 2.0.0",
-    "llama-index >= 0.10.0",
+    "argilla >= 2.0.0, < 3.0.0",
+    "llama-index >= 0.10.0, < 0.11.0",
```

Comment:
maybe this is a little bit too tight for the llama-index version constraints. I guess setting it to smaller than 1 would perhaps be better.